### PR TITLE
docs(stepfunctions-tasks): fix typo in EmrClusterScaleDownBehavior description

### PR DESCRIPTION
**Pull request by SaxyPandaBear**
It ain't much, but it's honest work -- I only noticed it cause I've been going down a rabbit hole of defining EMR cluster steps in CDK this week.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by BenChaimberg — CHANGES_REQUESTED** [review]
Well, if we're changing this anyway...can we make this description more informative? Maybe take a look at [the corresponding API docs](https://docs.aws.amazon.com/emr/latest/APIReference/API_RunJobFlow.html#EMR-RunJobFlow-request-ScaleDownBehavior) and see if you can pull something that gives a better description of what these values actually *do*.

**Comment by SaxyPandaBear** [comment]
@BenChaimberg Sounds good to me. I'll take a look

**Review by BenChaimberg — APPROVED** [review]

**Comment by SaxyPandaBear** [comment]
@BenChaimberg looks like it said the CodeBuild failed but I can't see why (but the github bot posted CodeBuild success?)

File: packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts [hunk 1]
```diff
@@ -360,8 +360,10 @@ export class EmrCreateCluster extends sfn.TaskStateBase {
 
 export namespace EmrCreateCluster {
   /**
-   * Valid valus for the Cluster ScaleDownBehavior
+   * The Cluster ScaleDownBehavior specifies the way that individual Amazon EC2 instances terminate when an automatic scale-in activity
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts:363]:
```suggestion
   * The Cluster ScaleDownBehavior specifies the way that individual Amazon EC2 instances terminate when an automatic scale-in activity
```

**Review by BenChaimberg — CHANGES_REQUESTED** [review]

**Review by BenChaimberg — APPROVED** [review]

**Pull request merged**